CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 * Use workspace dependencies for all dependencies ([#4132])
 
 ### Twoliter
-* Update bottlerocket-core-kit to v2.3.4 ([#4152], [#4143], [#4139])
+* Update bottlerocket-core-kit to v2.3.5 ([#4156], [#4152], [#4143], [#4139])
```

Comment:
I don't think a "Twoliter" section makes a lot of sense going forward; this is just a dependency update and should be filed under OS changes.